code/src/utils/validators/deliverySheme.ts

```diff
@@ -0,0 +1,9 @@
+import { z } from "zod";
+
+const DeliveryMethod = z.enum(["NOVA", "UKRPOSHTA"]);
```

Comment:
Get this values from `deliveryMethods` constants. Having single source of truth is always better approach

Reply:
created array of values